app/views/dashboards/followers.html.erb

```diff
@@ -31,7 +31,7 @@
               <span class="dashboard-username">@<%= user.username %></span>
               <a href="#"
                 id="dashboard-follow"
-                class="cta follow-action-button dashboard-follow"
+                class="cta follow-action-button dashboard-follow follow-user"
```

Comment:
[This indicates to us to user the bulk user endpoint](https://github.com/thepracticaldev/dev.to/pull/7676/files#diff-100ce16bf208667b69c5a3725bbe6a60R4)

Reply:
such a subtle fix :D